### PR TITLE
Adjust commands to use React selectors if element has isReactElement flag

### PR DESCRIPTION
**Pull request by baruchvlz**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

Correctly use `react$$` when waiting for React element to exist in DOM.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

Issue: #4442

### CODE REVIEW

File: packages/webdriverio/src/commands/element/isExisting.js [hunk 1]
```diff
@@ -41,5 +41,7 @@
  */
 
 export default function isExisting () {
```
Comment (CrispusDH) [inline packages/webdriverio/src/commands/element/isExisting.js]: Is it worth to cover other `isSomething` methods?

Comment (baruchvlz) [reply packages/webdriverio/src/commands/element/isExisting.js]: That's a good point, I'll take a look at how these methods and adjust them if needed.

**Comment by baruchvlz** [comment]
@christian-bromann Will do, I'll ping you on gitter if any questions arise.

File: packages/webdriverio/src/commands/element/isExisting.js (hunk 1 above)
Comment (baruchvlz) [reply packages/webdriverio/src/commands/element/isExisting.js]: The only other command I could find using this was `isDisplayed`. It has been adjusted in a50e10d .

**Pull request merged**